src/material/list/selection-list.spec.ts

```diff
@@ -1277,6 +1289,16 @@ class SelectionListWithDisabledOption {
 class SelectionListWithSelectedOption {
 }
 
+@Component({
+  template: `
+  <mat-selection-list>
+    <mat-list-option [selected]="true" [value]="itemValue">Item</mat-list-option>
+  </mat-selection-list>`
+})
+class SelectionListWithSelectedOptionAndValue {
```

Comment:
Seems like this is failing in the CI because it wasn't added to the `declarations`.